src/scripts/components/Accordion.ts

```diff
@@ -1,155 +1,215 @@
 import { ComponentElement } from '@locomotivemtl/component-manager';
 
-export default class Accordion extends HTMLElement {
-    static readonly DURATION = 300;
-    static readonly CLASS_OPEN = 'is-open';
-    private onClickBind: any;
-    private $root: HTMLDetailsElement;
-    private $summary: HTMLElement;
-    private $content: HTMLElement;
-    private $parent: HTMLElement | null;
-    private animation: Animation | null;
-    private isClosing: boolean;
-    private isExpanding: boolean;
+export class Accordion extends HTMLElement {
+    static CLASS_ACTIVE = 'is-active';
+    static EASING = 'cubic-bezier(0.33, 1, 0.68, 1)';
+    static DURATION = 300;
+
+    private $summary: HTMLElement | null = null;
+    private $content: HTMLElement | null = null;
+    private $details: HTMLDetailsElement | null = null;
+    private animation: Animation | null = null;
+    private isClosing: boolean = false;
```

Comment:
I recommend using the term `collapse` and `collapsing` instead of `close` and `closing` to match `expand` and `expanding` when it comes to animation-related functionality.